functions.php

```diff
@@ -952,7 +937,7 @@ function wp_autoupdates_is_plugins_auto_update_email_enabled() {
  * @return bool True if themes notifications are enabled, false otherwise.
  */
 function wp_autoupdates_is_themes_auto_update_email_enabled() {
-	$enabled = ! defined( 'WP_DISABLE_THEMES_AUTO_UPDATE_EMAIL' ) || ! WP_DISABLE_THEMES_AUTO_UPDATE;
+	$enabled = true;
```

Comment:
Instead of setting `$enabled` to true, would be cleaner to just pass `true` to the filter.

Reply:
And keep mentioning $enabled in the inline docs?

Reply:
yes